src/store/csv/stub/content/pupils-at-expected-standards/2023-03/1.csv

```diff
@@ -0,0 +1,1155 @@
+E09000006,Bromley,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2018-09-01T00:00:00/P1Y,75.0,Percentage,%,,,,1,
+E09000006,Bromley,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2019-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000006,Bromley,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2020-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2019-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2018-09-01T00:00:00/P1Y,67.0,Percentage,%,,,,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2017-09-01T00:00:00/P1Y,67.0,Percentage,%,,,,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2020-09-01T00:00:00/P1Y,,Percentage,%,,,x,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2021-09-01T00:00:00/P1Y,63.0,Percentage,%,,,,1,
+E09000008,Croydon,Local Authority District,"Percentage of pupils meeting the expected standard in reading, writing and maths by end of primary school (KS2)",Pupils at expected standards by end of primary school,Mission 5,Spreading opportunity and improving public services,2015-09-01T00:00
```

Comment:
These rows need to be fixed, several rows are spread out across two lines.

Reply:
completed